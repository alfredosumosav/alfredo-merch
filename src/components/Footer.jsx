import React from 'react'
import '../styles/components/Footer.css';

const Footer = () => (
    <div className="Footer">
      <p className="Footer-title">Alfredo Merch</p>
      <p className="copy">All Lefts Reserved</p>
    </div>
  )

export default Footer
