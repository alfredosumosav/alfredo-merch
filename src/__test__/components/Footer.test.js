/* eslint-disable no-undef */
import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';
import Footer from '../../components/Footer';

describe('<Footer />', () => {
  const footer = mount(<Footer />);

  test('Render Footer component', () => {
    expect(footer.length).toEqual(1);
  });

  test('Title Renders', () => {
    expect(footer.find('.Footer-title').text()).toEqual('Alfredo Merch');
  });
});

describe('Footer Snapshot', () => {
  test('Footer renders correctly', () => {
    const footer = create(<Footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  });
});
