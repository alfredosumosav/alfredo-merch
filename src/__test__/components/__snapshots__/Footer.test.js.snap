// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot Footer renders correctly 1`] = `
<div
  className="Footer class"
>
  <p
    className="Footer-title"
  >
    Alfredo Merch
  </p>
  <p
    className="copy"
  >
    All Lefts Reserved
  </p>
</div>
`;
